### ISSUE TITLE
[MSTeams][Accessibility of Adaptive cards on Mobile] [Random display of Adaptive Cards on Mobile]

### ISSUE THREAD

**Issue by srishtisatwah** [microsoft/AdaptiveCards#3883]
# Platform

* Android
* iOS

# Author or host

Rendering adaptive cards and sending to MS Teams

# Details
We are **using MS Teams developer preview to display adaptive cards (carousal).**
We have made sure the json we send doesn't have empty strings and properties not supported by Teams. It has been woking fine on Desktop. 
However, we notice that **order of cards being displayed on mobile is incorrect.**

**In the attached pictures, notice the dots below, card #1 is in the second slot while #3 is in the first slot. 
Ideally, the cards should be in the respective 1st and 3rd slot.**

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/10636538/77787117-3b080d00-701c-11ea-9fb5-91b43812cd25.png)
![MicrosoftTeams-image](https://user-images.githubusercontent.com/10636538/77787124-3d6a6700-701c-11ea-851e-1fa73f40872c.png)

**Comment by nesalang** [comment]
@nesalang 
Ask team about carousal feature

**Comment by shalinijoshi19** [comment]
@nesalang FYI assigned this to you per our conversation. @siduppal / @VikrantSingh01 FYI in case we need help here (i think @nesalang had questions about the carousel feature?)

**Comment by nesalang** [comment]
@Aditya0603 

Hi Aditya,
Can you check this issue? AdaptiveCards SDK doesn't support carousal feature, it would be nice to k now how this feature supposed to work.

Thank you

**Comment by jwoo-msft** [comment]
@shalinijoshi19 
Can you forward this issue to Teams?

**Comment by adumrewal** [comment]
@srishtisatwah please share the build number on which you faced this issue?
A fix for this has been rolled out recently, can you please try the latest Teams iOS app and let us know if you are still facing the issue?

**Comment by Subhasish-msft** [comment]
closing this issue, @sasatpat , please reopen it, if you are still facing the issue
